### PR TITLE
various fixes for headset state and documentation

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

File: react-app/src/library/services/vendor-implementations/jabra/jabra.ts [hunk 1]
```diff
@@ -233,7 +233,7 @@ export default class JabraService extends VendorImplementation {
     try {
       this.callLock = await this.callControl.takeCallLock();
       if (this.callLock) {
-        this.activeConversationId = callInfo.conversationId;
+        this.pendingConversationId = callInfo.conversationId;
```
Comment (maxwellmooney13) [inline react-app/src/library/services/vendor-implementations/jabra/jabra.ts:236]: See above comment

File: react-app/src/library/services/vendor-implementations/jabra/jabra-native/jabra-native.ts [hunk 2]
```diff
@@ -196,8 +196,7 @@ export default class JabraNativeService extends VendorImplementation {
   }
 
   async outgoingCall(callInfo: CallInfo): Promise<void> {
-    // this.pendingConversationId = callInfo.conversationId;
-    this.activeConversationId = callInfo.conversationId;
+    this.pendingConversationId = callInfo.conversationId;
```
[the remark this answers is not in the document]
Comment (jensengar) [reply react-app/src/library/services/vendor-implementations/jabra/jabra-native/jabra-native.ts:199]: So the problem with using active is we still get a `offHook` event whether it's outgoing or incoming. At that point, the `pendingConversationId` is assigned to the `activeConversationId`. If we set `activeConversationId` here, it gets overridden to `null` when the `offhook` event comes in.

**Review by maxwellmooney13 — APPROVED** [review]

File: react-app/src/library/types/consumed-headset-events.ts [hunk 3]
```diff
@@ -11,7 +11,6 @@ export enum HeadsetEvents {
     loggableEvent = 'loggableEvent',
     webHidPermissionRequested = 'webHidPermissionRequested',
     deviceConnectionStatusChanged = 'deviceConnectionStatusChanged',
-    deviceMuteChanged = 'deviceMuteChanged',
```
Comment (djhouseknecht) [inline react-app/src/library/types/consumed-headset-events.ts:14]: Is it ok to just remove this? It does need to be renamed like the other event?

Comment (jensengar) [reply react-app/src/library/types/consumed-headset-events.ts:14]: there's a `deviceMuteStatusChanged` event already. We were using `deviceMuteChanged` for mutes and `deviceHoldStatusChanged` for holds. The `deviceMuteStatusChanged` seemed to be unused so I switched over to that one to match the naming style for holds and deleted the old one so we only have a single mute event in the types.

Comment (djhouseknecht) [reply react-app/src/library/types/consumed-headset-events.ts:14]: 👍🏼

**Review by djhouseknecht — APPROVED** [review]

**Pull request merged**